src/languages/lib/css-shared.js

```diff
@@ -30,6 +30,10 @@ export const MODES = (hljs) => {
         '|dpi|dpcm|dppx' +
         ')?',
       relevance: 0
+    },
+    CSS_VARIABLE: {
+      className: "attr",
+      begin: /--[A-Za-z][A-Za-z0-9_-]*/
```

Comment:
[note] @joshgoebel I thought it would make sense to place this under `lib/css-shared.js` for DRY.